### PR TITLE
iOS Dynamic Framework improvements

### PR DESCRIPTION
**Pull request by marmelroy**
Hello,

I am using Keyframes in a Swift-only project and encountered a number of issues when integrating it as a dynamic framework - the most significant being the fact that the project didn't have a public header. 

This PR solves the following:
- Adding a public header to improve usage as a dynamic framework. 
- Fixing the example app so that it doesn't need a bridging header. 
- Fixing a missing init warning on KFVectorView.
- More controversially perhaps, renaming the framework to the correct case (i.e. Keyframes instead of keyframes).

### CODE REVIEW

File: ios/keyframes/src/Keyframes.h [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+/**
+ * This file is generated using the remodel generation script.
+ * The name of the input file is KFVector.value
+ */
+
```
Comment (LazyChild) [inline ios/keyframes/src/Keyframes.h]: remove this please, it's not part of the header

File: keyframes.podspec [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+  spec.name             = 'Keyframes'
   spec.version          = '1.0.0'
   spec.license          = { :type => 'BSD', :file => 'LICENSE' }
   spec.homepage         = 'https://github.com/facebookincubator/Keyframes'
   spec.author           = { 'Sean Lee' => 'landasiastudio@gmail.com', 'Renyu Liu' => 'sd.qd.lry@gmail.com' }
   spec.summary          = 'Vector+keyframe rendering framework for iOS'
   spec.source           = { :git => 'https://github.com/facebookincubator/Keyframes.git', :tag => "v#{spec.version}" }
   spec.source_files     = 'ios/keyframes/src/**/*.{h,m,mm,cpp}'
-  spec.public_header_files = 'ios/keyframes/src/{DataModel/KFVector,Layers/KFVectorLayer,ParsingHelpers/KFVectorParsingHelper,Views/KFVectorView,Helpers/KFUtilities}.h'
+  spec.public_header_files = 'ios/keyframes/src/{keyframes,DataModel/KFVector,Layers/KFVectorLayer,ParsingHelpers/KFVectorParsingHelper,Views/KFVectorView,Helpers/KFUtilities}.h'
```
Comment (LazyChild) [inline keyframes.podspec]: Keyframes?

File: ios/keyframes/src/ParsingHelpers/KFVectorParsingHelper.h [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 #import <UIKit/UIKit.h>
 
-#import <keyframes/KFUtilities.h>
+#import "KFUtilities.h"
```
Comment (LazyChild) [inline ios/keyframes/src/ParsingHelpers/KFVectorParsingHelper.h]: why?

File: ios/keyframes/src/Keyframes.h [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+#ifndef Keyframes_H
+#define Keyframes_H
+
```
Comment (LazyChild) [inline ios/keyframes/src/Keyframes.h]: do we need this if we are using `#import`?

File: ios/keyframes/src/ParsingHelpers/KFVectorParsingHelper.h (hunk 3 above)
Comment (marmelroy) [inline ios/keyframes/src/ParsingHelpers/KFVectorParsingHelper.h]: When using``` #import <keyframes/KFUtilities.h>```, Xcode generates an "include of non-modular header inside framework module 'Keyframes.KFVectorParsingHelper' error at the point of use (despite KFUtilities.h being clearly marked as a public header). This change fixes it.

**Comment by marmelroy** [comment]
Thanks for the quick feedback @LazyChild. I made three of the four requested changes.

File: ios/keyframes/src/ParsingHelpers/KFVectorParsingHelper.h (hunk 3 above)
Comment (LazyChild) [inline ios/keyframes/src/ParsingHelpers/KFVectorParsingHelper.h]: I don't see this issue on my side tho. if KFUtilities.h is public, there shouldn't be this error right? Do you mind to try it again and maybe attach a screenshot?

**Comment by marmelroy** [comment]
So the real issue was the way that the Swift sample project was set up. It's now fixed and the PR uses 
```#import <Keyframes/[public class].h>``` everywhere.

**Pull request merged**